carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/device/Device.java

```diff
@@ -818,7 +818,7 @@ public File generateUiDump(String screenshotName) {
         // TODO: investigate how to connect screenshot with xml dump: screenshot
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1135)